libs/java/server_common/src/test/java/com/yahoo/athenz/common/server/cert/X509CertRecordTest.java

```diff
@@ -45,6 +45,7 @@ public void testX509CertRecord() {
         certRecord.setLastNotifiedTime(now);
         certRecord.setExpiryTime(now);
         certRecord.setSvcDataUpdateTime(now);
+        certRecord.setSiaProvider("SIA-EKS 1.52.0");
```

Comment:
let's update this test case to use cert.setSiaProvider("EKS") to be consistent with the implementation.